modules/ROOT/pages/6.5-release-notes.adoc

```diff
@@ -71,6 +71,16 @@ For information on using premium skins and icon packs, see: xref:premium-skins-a
 
 {productname} 6.5 also includes the following bug fixes:
 
+=== Inserting two tables consecutively without focus in the editor resulted in the second table being inserted at the wrong position.
+//#TINY-3909
+
+In previous versions of {productname}, an issue was identified that resulted in the incorrect execution of the `remove_trailing_br` function by **DomParser**. This resulted in the insertion point focussing on the `<tr>` element instead of the `<td>` element.
```

Comment:
```suggestion
In previous versions of {productname}, an issue was identified that resulted in the incorrect execution of the `remove_trailing_br` function by **DomParser**. This resulted in the insertion point focusing on the `<tr>` element instead of the `<td>` element.
```